src/etc/inc/filter.inc

```diff
@@ -446,6 +446,12 @@ function filter_configure_sync($verbose = false)
         $limitrules .= "set limit table-entries {$config['system']['maximumtableentries']}\n";
     }
 
+    if ($config['system']['rulesoptimization'] <> "") {
```

Comment:
Thanks, let me rework the diff and submit a new one. I think that I also better rename it to rulesetoptimization for consistency.